diff --git a/libsqlite3/libsqlite3/sqlite3.c b/libsqlite3/libsqlite3/sqlite3.c
index 09f3e4a..3c3ef64 100644
--- a/libsqlite3/libsqlite3/sqlite3.c
+++ b/libsqlite3/libsqlite3/sqlite3.c
@@ -146544,6 +146544,7 @@ SQLITE_PRIVATE void sqlite3Pragma(
         if( tableSkipIntegrityCheck(pTab,pObjTab) ) continue;
         if( HasRowid(pTab) ) cnt++;
         for(nIdx=0, pIdx=pTab->pIndex; pIdx; pIdx=pIdx->pNext, nIdx++){ cnt++; }
+        (void)nIdx; /* Unused. */
       }
       if( cnt==0 ) continue;
       if( pObjTab ) cnt++;
